src/components/TopicTypeIcon.vue

```diff
@@ -11,7 +11,10 @@
 <template>
   <div
     class="TopicTypeIcon"
+    :class="{ 'navigator-icon': true, 'colorful-navigator-icon': shouldUseColorfulIcons && hasSymbolText }"
```

Comment:
I don't think we should always add the `"navigator-icon"` class here since this component isn't only ever used in the navigator. In fact, I believe the navigator component where this component is utilized is _already_ conditionally applying this exact class, so this is actually getting duplicated in that scenario.